website/docs/docs/dbt-cloud/cloud-configuring-dbt-cloud/connecting-gitlab.md

```diff
@@ -0,0 +1,49 @@
+---
+title: "Connecting your GitLab Account"
+id: "connecting-gitlab"
+sidebar_label: "Connecting your GitLab Account"
+---
+
+:::info Enterprise Feature
+
+This guide describes a feature of the dbt Cloud Enterprise plan. If you’re interested in learning more about an Enterprise plan, let us know at sales@getdbt.com.
+
+:::
+
+## Overview
+
+Connecting your GitLab account to dbt Cloud unlocks exciting and compelling functionality in dbt Cloud. Once your GitLab account is connected, you can:
+- trigger CI builds when Pull Requests are opened in GitLab!
+- log into dbt Cloud via oauth through GitLab!
+- add new repos to Cloud in a single click!
+- and more!
+
+## Linking dbt Cloud to your GitLab account
+
+To link your dbt Cloud account to your GitLab account, navigate to your Account Settings and click the Integrations tab. 
+
+<Lightbox src="/img/docs/dbt-cloud/cloud-configuring-dbt-cloud/gitlab_navigation_setup.gif" title="Like so!"/>
+
+Here you can link your GitLab account to dbt Cloud. 
+
+You'll need to set up an application at GitLab in order to make it official - [they have a guide for that here](https://docs.gitlab.com/ee/integration/oauth_provider.html#adding-an-application-through-the-profile)! For your dbt Cloud application, you'll want to check the `api` scope level.
+
+Once you've gotten your Application ID and Secret from GitLab, you can click the Edit button in your Integrations tab, and copy-paste those values into the form.
+
+<Lightbox src="/img/docs/dbt-cloud/cloud-configuring-dbt-cloud/gitlab_integration_edit_button.png" title="This button!"/>
+
+GitLab will then ask for your explicit approval:
+
+<Lightbox src="/img/docs/dbt-cloud/cloud-configuring-dbt-cloud/gitlab_application_auth.png"/>
+
+And then you're all set!
+
+## Usage Notes
+
+Note that only the GitLab admin who sets up new dbt Cloud projects needs to connect their GitLab account. It’s not necessary for every dbt Cloud user to connect GitLab (but it can be convenient to do so!)
```

Comment:
Not so! Every Developer who wishes to use the IDE will need to OAuth with GitLab. Only the GitLab admin will need to manage the integration though

Reply:
Can we in fact add a section here around using the IDE? You'll see modals that prompt you to log in with GitLab if you have not logged in with GitLab and you try developing for the first time

Reply:
I don't think I follow, can we discuss this more?

I think I need a test account, or the ability to spin up test accounts 😅

Reply:
All set!